parquet-hadoop/README.md

```diff
@@ -501,3 +501,11 @@ If `false`, key material is stored in separate new files, created in the same fo
 **Description:** Length of key encryption keys (KEKs), randomly generated by parquet key management tools. Can be 128, 192 or 256 bits.  
 **Default value:** `128`
 
+---
+
+**Property:** `parquet.hadoop.vectored.io.enabled`  
+**Description:** Flag to enable use of the FileSystem Vector IO API on Hadoop releases which support the feature.
+If `true` then an attempt will be made to dynamically load the relevant classes; 
```

Comment:
Does this feature need to upgrade NameNode and DataNode?

Reply:
no, hdfs doesn't support it. Native IO does, so if you use file:// URLS you get direct NIO vectored IO into buffers (yay! hadoop APIs move to the 2010s!). S3A supports it with multiple parallel GET with some range coalescing in between. Would love ABFS connector to support it too...